python/grass/jupyter/interactivemap.py

```diff
@@ -305,15 +308,41 @@ def _import_ipyleaflet(error):
         # Store Region
         self.region = None
 
-        if self._ipyleaflet:
-            basemap = xyzservices.providers.query_name(tiles)
-            if API_key and basemap.get("accessToken"):
-                basemap["accessToken"] = API_key
-            layout = self._ipywidgets.Layout(width=f"{width}px", height=f"{height}px")
-            self.map = self._ipyleaflet.Map(
-                basemap=basemap, layout=layout, scroll_wheel_zoom=True
+        if crs:
```

Comment:
You need to get the CRS of the current project with g.proj, use the 'srid' key to get EPSG